### ISSUE TITLE
Can't parse meta-tag with data-i18n attribute

### ISSUE THREAD

**Issue by alexshurik** [i18next/i18next-parser#94]
- Is this a bug or a feature request? Bug
- What version are you running (`0.x` or `1.x`)? 1.0.0-beta12
- Which runner (cli, gulp...)? cli

## What I'm trying to do?

*Parse HTML file with meta-tags with data-i18n attribute such as* `<meta data-i18n="site-content" name="content">` 

## How am I doing it?

```
./node_modules/i18next-parser/bin/cli.js 'src/scripts/**/*.{js,jsx}' 'index.html' -c i18next-parser-config.js
```

Here is my config file:

```
module.exports = {
    reactNamespace: true,
    locales: ['ru', 'en'],
    output: 'src/scripts/i18n',
    createOldCatalogs: false,
    defaultNamespace: 'html',
    keySeparator: false,
    namespaceSeparator: ':',
    keepRemoved: true
};
```

## What was the result?

*Parsers just stops at the moment of parsing HTML. Example of output*

```
> ./node_modules/i18next-parser/bin/cli.js 'src/scripts/**/*.{js,jsx}' 'index.html' -c i18next-parser-config.js


  i18next Parser
  --------------
  Input:  src/scripts/**/*.{js,jsx}, index.html
  Output: src/scripts/i18n

  [read]  /Users/admin/Projects/38crew-site/frontend/index.html
```

Then scripts just stops at the last line and nothing happens after that

## Thoughts and experiments:

But everything will work as expected, if i add closing meta-tag:
`<meta data-i18n="site-content" name="content"></meta>`

**Comment by karellm** [comment]
Can you please provide the full HTML file or a failing test case? Thanks!

**Comment by alexshurik** [comment]
I can not understand what the problem is

For example this code doesn't work:
```
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8"/>
    <meta name="description"  data-i18n="key228" content="content"/>
    <meta name="keywords" content="some keywords"/>
    <meta name="google-site-verification" content="google-key"/>
    <meta name="yandex-verification" content="yandex-key"/>
    <title>Lorem ipsum dolor sit amet, consectetur adipiscing elit</title>
</head>

<body id="main"></body>
</html>
```

This works but takes about 10 seconds (shorter text in title):
```
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8"/>
    <meta name="description"  data-i18n="key228" content="content"/>
    <meta name="keywords" content="some keywords"/>
    <meta name="google-site-verification" content="google-key"/>
    <meta name="yandex-verification" content="yandex-key"/>
    <title>Lorem ipsum dolor</title>
</head>

<body id="main"></body>
</html>
```

And this works instantly (no <mata name="yandex-verification content="yandex-key"/> tag
```
<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8"/>
    <meta name="description"  data-i18n="key228" content="content"/>
    <meta name="keywords" content="some keywords"/>
    <meta name="google-site-verification" content="google-key"/>
    <title>Lorem ipsum dolor</title>
</head>

<body id="main"></body>
</html>
```

**Comment by karellm** [comment]
I just published `1.0.0-beta14` which fixes this issue. The old lexer wasn't support self-closing tags
